### ISSUE TITLE
Message for merging configuration

### ISSUE THREAD

**Issue by tulinkry** [OpenGrok/OpenGrok#1479]
After changes in #1433 (user can send new configuration to the web app through the Messages tools) it would be quite useful if user could control if the new configuration will merge with the current or will overwrite it (current behaviour).

**Comment by tulinkry** [comment]
Actually it could be useful to even have a possibility to set some primitive properities directly with something like
```bash
./Messages -n config -t set "messageLimit = 1000"
./Messages -n config -t set "userPage = http://example.com?user="
```

**Comment by tulinkry** [comment]
The second part is implemented in 662c4211d55a90a4b0555965358f6508ca9cedeb

**Comment by tulinkry** [comment]
Closing as the first part is sort of implemented by #1590 where user can merge several configuration into a single one and then send it as a single file.
